src/containers/teleport/components/dataIbcHistory/compDataHstory.tsx

```diff
@@ -0,0 +1,110 @@
+import { Coin } from '@cosmjs/launchpad';
+import { useMemo } from 'react';
+import { useIbcDenom } from 'src/contexts/ibcDenom';
+import { CYBER, PATTERN_CYBER } from 'src/utils/config';
+import { getDisplayAmount, trimString } from 'src/utils/utils';
+import fromToIbc from 'images/fromToIbc.svg';
+import { Account, DenomArr, FormatNumberTokens } from 'src/components';
+import upArrow from 'images/up-arrow.png';
+import downArrow from 'images/down-arrow.png';
+import complete from 'images/statusTx/complete.png';
+import pending from 'images/statusTx/pending.png';
+import timeout from 'images/statusTx/timeout.png';
+import refunded from 'images/statusTx/refunded.png';
+import { TxsType } from '../../type';
+import { HistoriesItem, StatusTx } from '../../ibc-history/HistoriesItem';
+import styles from './styles.module.scss';
+
+const mapStatusTxImg = {
+  [StatusTx.COMPLETE]: complete,
+  [StatusTx.PENDING]: pending,
+  [StatusTx.TIMEOUT]: timeout,
+  [StatusTx.REFUNDED]: refunded,
+};
+
+export function TypeTsx({ sourceChainId }: { sourceChainId: string }) {
+  if (sourceChainId === CYBER.CHAIN_ID) {
+    return (
+      <div className={styles.containerTypeTsx}>
+        <img src={upArrow} alt="upArrow" />
+        <div>{TxsType.Withdraw}</div>
+      </div>
+    );
+  }
+
+  return (
```

Comment:
need to have  1 return, and check inside it.
for this good to create variable as `isCyberChain`